zoo/resources/templates/dependency_overview.html

```diff
@@ -0,0 +1,106 @@
+{% extends 'base.html' %}
+{% load utils %}
+
+{% block menu %}
+  {% with selected='resources' %}
+    {{ block.super }}
+  {% endwith %}
+{% endblock %}
+
+{% block navtitle %}
+{% url 'library_overview' as library_overview_url %}
+{% url 'dependency_overview' as dependency_overview_url %}
+{% url 'language_overview' as language_overview_url %}
+
+<nav class="ui container secondary pointing menu">
+  <a
+    href="{{ library_overview_url }}"
+    class="{% if request.path == library_overview_url or request.path == dependency_overview_url %}active {% endif %}item">
+    <i class="book icon"></i>
+    Libraries
+  </a>
+  {% comment %} <a href="{% url 'docker_template_overview' %}"
```

Comment:
This is probably WIP, just reminder to remove the comments before the final version.